signing_service/signing_service/signing_service.py

```diff
@@ -581,7 +582,10 @@ def main() -> None:
     logging.config.fileConfig(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'logging.ini'))
 
     # Parse required arguments.
-    args = _parse_arguments()
+    try:
```

Comment:
this seems unneeded, please remove

Reply:
I understood form @cameel comment on the issue, that we don't want stacktrace, but it might not be the case?

Reply:
Done